<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemapDialog</class>
 <widget class="QDialog">
  <property name="windowTitle">
   <string>Remap</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>From Colors:</string>
        </property>
        <property name="toolTip">
         <string>Palette index (range) of the colors to be replaced or 256 if transparent</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LineEditWidget" name="colorFrom0LineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>(Palette-)Index of the first color</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="minimumWidth">
           <number>8</number>
          </property>
          <property name="maximumWidth">
           <number>8</number>
          </property>
          <property name="text">
           <string notr="true">-</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEditWidget" name="colorFrom1LineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>(Palette-)Index of the last color</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>To Colors:</string>
        </property>
        <property name="toolTip">
         <string>Palette index (range) of the replacement color or 256 if transparent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LineEditWidget" name="colorTo0LineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>(Palette-)Index of the first replacement color</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="minimumWidth">
           <number>8</number>
          </property>
          <property name="maximumWidth">
           <number>8</number>
          </property>
          <property name="text">
           <string notr="true">-</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEditWidget" name="colorTo1LineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>(Palette-)Index of the last replacement color</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Range:</string>
        </property>
        <property name="toolTip">
         <string>The frames which needs to be changed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LineEditWidget" name="range0LineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>The first frame to be changed</string>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel">
          <property name="minimumWidth">
           <number>8</number>
          </property>
          <property name="maximumWidth">
           <number>8</number>
          </property>
          <property name="text">
           <string notr="true">-</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEditWidget" name="range1LineEdit">
          <property name="charWidth">
           <number>5</number>
          </property>
          <property name="toolTip">
           <string>The last frame to be changed</string>
          </property>
          <property name="text">
           <string notr="true">0</string>
          </property>
          <property name="alignment">
           <enum>Qt::AlignCenter</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="remapButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remapCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="placementButtonGroup"/>
 </buttongroups>
</ui>
